src/platform/site-wide/va-footer/containers/Main.jsx

```diff
@@ -37,10 +41,7 @@ export class Main extends React.Component {
       },
       false,
     );
-
-    this.setState({
```

Comment:
can't set state in `componentDidMount`- moving this into the constructor didn't impact the behavior.